testing/trino-tests/src/test/java/io/trino/tests/TestServer.java

```diff
@@ -257,25 +258,21 @@ public void testVersionOnError()
     @Test
     public void testVersionOnCompilerFailedError()
     {
-        int numberColumns = 170;
         String tableName = "memory.default.test_version_on_compiler_failed";
         try (TestingTrinoClient testingClient = new TestingTrinoClient(server, testSessionBuilder().build())) {
             testingClient.execute("DROP TABLE IF EXISTS " + tableName);
-            testingClient.execute("CREATE TABLE " + tableName + " AS SELECT " +
-                    IntStream.range(0, numberColumns)
-                            .mapToObj(columnNumber -> format("'' AS f%s", columnNumber))
-                            .collect(joining(", ")));
-
-            String pivotQuery = "SELECT x, y " +
-                    "FROM " + tableName + " " +
-                    "CROSS JOIN UNNEST(" +
-                    IntStream.range(0, numberColumns)
-                            .mapToObj(Integer::toString)
-                            .collect(joining(", ", "ARRAY[", "]")) + "," +
-                    IntStream.range(0, numberColumns)
-                            .mapToObj(columnNumber -> format("f%s", columnNumber))
-                            .collect(joining(", ", "ARRAY[", "]")) + ") t(x, y)";
-            checkVersionOnError(pivotQuery, "TrinoException: Compiler failed(?s:.*)at io.trino.sql.gen.PageFunctionCompiler.compileProjection");
+            testingClient.execute("CREATE TABLE " + tableName + " AS SELECT '' as foo");
```

Comment:
I guess the query can select from VALUES and thus you don't need to DROP and CREATE here

Reply:
Using values is dangerous as it can be optimized away.  I'm just going to leave the query as is.